### ISSUE TITLE
Citizens frozen in place 

### ISSUE THREAD

**Issue by kezmodius** [ldtteam/minecolonies#10440]
### Is there an existing issue for this?

- [X] I have searched the existing issues.

### Are you using the latest MineColonies Version?

- [X] I am running the latest **beta/release** version of MineColonies for my Minecraft version.
I am also running the latest versions of other mods that are part of my problem.

### Did you check on the Wiki? or ask on Discord?

- [X] I checked the MineColonies Wiki and made sure my issue is not covered there. Or I was sent from discord to open an issue here.

### What were you playing at the time? Were you able to reproduce it in both settings?

- [X] Single Player
- [X] Multi Player

### Minecraft Version

1.20

### MineColonies Version

1.20.1-1.1.730-snapshot

### Structurize Version

1.20.1-1.0.742-release

### Related Mods and their Versions

Forge 47.3.0
**Modlist**: 
Sa(N)ta(N) - Yuletide of Madness (by Nitespring)
The Fence Unleashed (by TheMrMilchmann)
Patchouli (by Vazkii)
Camps. Castles. Carriages. (by Eugene2000)
Bookshelf (by DarkhaxDev)
CreativeCore (by CreativeMD)
Snowy Tents (by SuperWarioModTeam)
TownTalk (by Raycoms)
JourneyMap (by techbrew)
FramedBlocks (by XFactHD)
Medieval Music[Forge/Fabric] (by someaddon)
Snow! Real Magic! ⛄ (NeoForge) (by Snownee)
Cosmetic Armor Reworked (by LainMI)
Vanilla Plus Tools (by Raycoms)
Collective (by Serilum)
Polymorph (Fabric/Forge/Quilt) (by TheIllusiveC4)
Mouse Tweaks (by YaLTeR)
Fir Forest Camping (by ChoccoMilk_)
Farsight [Forge/Neo] (by someaddon)
Cupboard (by someaddon)
Curios API (Forge/NeoForge) (by TheIllusiveC4)
Better Fps - Render Distance[Forge] (by someaddon)
Structurize (by Raycoms)
Better chunk loading[Forge/Fabric] (by someaddon)
Just Enough Items (JEI) (by mezz)
Structure Gel API (by ModdingLegacy)
Framework (by MrCrayfish)
Jade 🔍 (by Snownee)
Balm (by BlayTheNinth)
TrashSlot (by BlayTheNinth)
Cobweb (by CrystalSpider)
Waystones (by BlayTheNinth)
Multi-Piston (by Raycoms)
Dynamic View[Forge] (by someaddon)
AFK Command (by aylanj123)
Domum Ornamentum (by OrionOnline)
MineColonies (by Raycoms)
Mc2Discord (by DenisD3D_)
Better Villages - Forge (by jtl_elisa)
Botany Pots (by DarkhaxDev)
WorldEdit (by sk89q)
Iron Chests (by ProgWML6)
Repurposed Structures (Neoforge/Forge) (by telepathicgrunt)
Charm of Undying (Fabric/Forge/Quilt) (by TheIllusiveC4)
Corpse (by henkelmax)
Maiden's Merrymaking - A Minecraft Christmas and Winter Holiday Mod (by Mama_Michelle)
Entity Culling Fabric/Forge (by tr7zw)
Cloth Config API (Fabric/Forge/NeoForge) (by shedaniel)
spark (by Iucko)
Botany Trees (by DarkhaxDev)
Maiden's Holiday Horticulture--Holiday Flowers and Plants (by Mama_Michelle)
Structure Essentials[Forge/Fabric] (by someaddon)
ChristmasColonies (by Raycoms)
Oculus (by Asek3)
Comforts (Fabric/Forge/Quilt) (by TheIllusiveC4)
AmbientSounds 6 (by CreativeMD)
Paxi (Forge) (by YUNGNICKYOUNG)
SuperMartijn642's Core Lib (by SuperMartijn642)
AppleSkin (by squeek502)
Library Ferret - Forge (by jtl_elisa)
BlockUI (by Raycoms)
River Redux (by Jasmine)
YUNG's API (Forge) (by YUNGNICKYOUNG)
Eternal Winter (by ichttt)
Packed Up (Backpacks) (by SuperMartijn642)
Tree Harvester (by Serilum)
Snowy Spirit (by MehVahdJukaar)
Recipe Essentials[Forge/Fabric] (by someaddon)
Server Performance - Smooth Chunk Save[Forge/Fabric] (by someaddon)
Moonlight Lib (by MehVahdJukaar)
Lootr (Forge & NeoForge) (by Noobanidus)
Decorative Winter (by lilypuree)
Festive Delight (by To0pa)
Limited Chunkloading - Chunk cleanup[Forge/Fabric] (by someaddon)
Kiwi 🥝 (NeoForge) (by Snownee)
SuperMartijn642's Config Lib (by SuperMartijn642)
Farmer's Delight (by vectorwing)
permafrost biome (by tombkingsts)
fix GPU memory leak[Forge/Fabric] (by someaddon)
Leaky - Item Lag Fix[Forge/Fabric] (by someaddon)
JadeColonies (by uecasm)
Connectivity[Forge/Fabric] (by someaddon)
Just Another Rotten Flesh to Leather Mod (by alexdaum1)
Jade Addons (NeoForge) (by Snownee)
Better Snowball (by skniro)
Corail Woodcutter (by Corail_31)
Decorative Blocks (by stohun)
Snow Under Trees (by bl4ckscor3)
GeckoLib (by Gecko)
Chunk Sending[Forge/Fabric] (by someaddon)
Waddles (by Girafi)
Embeddium (by FiniteReality)
FerriteCore ((Neo)Forge) (by malte0811)
L2 Library (by lcy0x1)
TerraBlender (Forge) (by TheAdubbz)
Hostile Villages[Forge/Fabric] (by someaddon)

**Mods updated in the update**
creativecore v2.12.22
entityculling 1.71
farmer's delight 1.2.5
moonlight 2.13.25

**Mods added in the update**
Botany Pots Trees 9.0.17
charm of undying 6.5.0
cosmetic armour reworked v1a
iron chest 14.4.4
jrftl 1.6.0
vanillaplustools
thefenceunleashed
trashslot

### Current Behavior

Modpack updated on server. Joined server and all colonists are frozen in place. They are interactable normally when clicked on. They are recallable to TH and hutblock. They are still doing voicelines. When logging in the builder's messages came up in chat. They seem to be functioning apart from their lack of movement. They aren't invisible and working though, as nothing is being done.

### Expected Behavior

Colonists to move around and do their jobs.

### Reproduction Steps

I don't know how to replicate as I made an SP game and started a colony and they moved fine. I did not have a pre-existing world to test with.

### Logs

I will ask for server logs when owner is awake.

### Anything else?

I reset the request system and tried pause and restart, with no change. I am waiting for the server owner to restart the server to see if that helps, but someone else reported the same issue on discord and they said on server restart the colonists ran to the hutblocks and remained frozen there.

### Footer

---
## Viewers
* Add a thumbs-up to the bug report if you are also affected. This helps the bug report become more visible to the team and doesn't clutter the comments.
* Add a comment if you have any insights or background information that isn't already part of the conversation.

**Comment by ncottrill** [comment]
We're running ATM 10 modpack v. 10-1.18
Minecolonies v. 1.1.787 for 1.21.1
Our citizens were working fine for a few hours and then they froze. I do not know the condition that caused this. But, none are moving in any of the colonies on the multiplayer server.
We rebooted our server. The colonists moved for 1-2 minutes and then froze in place again.
Eagerly awaiting a fix.

**Comment by kezmodius** [comment]
Further comment: on my personal server running modpack https://www.curseforge.com/minecraft/modpacks/minecolonies-dragon-epoch with Minecolonies 1.1.729. I have the same problem. All citizens are frozen BUT they _were_ moving when I logged in, as a build finished and I saw a courier picking items up from a stash.
Edit to add: I looked through the server log and there was no mention of Minecolonies while I was logged in.

**Comment by WoopGuy** [comment]
I also encountered this problem
Citizens just stand frozen in singleplayer world, after reboot world, colonists moved for 30 sec. and then froze in place again.

I tried to find when this problem started. Version 1.1.786 where it started.

Mod list:
Additional Lights
 v. 2.1.9
Amplified Nether v. 1.2.5
AppleSkin v. 3.0.5
Architectury v,13.0.6
Artifacts v. 12.0.5
Better Advancments v. 0.4.2.19
BetterF3 v.11.0.2
Bookshelf v.21.1.12
Caelus API v.7.0.1
CarryOn v.2.2.2
Chunk Loader v.1.2.8
Cloth Config v.15.0.140
Curious API v.9.0.14
Deeper and Darker v.1.3.1
Domum Ornamentum v.1.0.204
Elytra Slot v.9.0.2
Elytra Trims v.3.5.4
EnchantmentDescriptions v.21.1.4
Ender dragon fight remastered v.5.0
EMF v.2.2.6
ETF v.6.2.5
ExpandAbility v.12.0.0
EXPcounter
FallingTree
Forgified Fabrick API Base
Friends and foes v.3.0.5
Gravestone Mod v.1.0.24
Handcraft v.4.0.2
Iris
Jade
JameLib
JEI
Kotlin for Forge
Lithium
Llama steeds
Minecolonies v.1.1.787
Multi-Piston v1.2.51
NeoForge 21.1.65
PrickleMC
Puzzles Lib
Resourceful Lib
Right Click Harvest
Scarecrows' Territory
Sodium
Sophisticated Backpacks
Sophisticated Core
Sophisticated Storage
Spyglass Improvements
Structurize v.1.0.754
SuperMartijn642's Config Lib
SuperMartijn642's Core Lib
Tool Trims
TownTalk
Trading Post
Trims Effects
Visual Workbench


Log file:
https://gist.github.com/WoopGuy/be56fc9b5f5ae807bd0d08b39cedcb02
[latest.log](https://github.com/user-attachments/files/17729001/latest.log)
